--- a/Pymacs/Rebox/rebox.py
+++ b/Pymacs/Rebox/rebox.py
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# -*- coding: latin-1 -*-
 # Copyright © 1991-1998, 2000, 2002 Progiciels Bourbeau-Pinard inc.
 # François Pinard <pinard@iro.umontreal.ca>, April 1991.
 
--- a/Pymacs/__init__.py
+++ b/Pymacs/__init__.py
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# -*- coding: latin-1 -*-
 # Copyright © 2002, 2003 Progiciels Bourbeau-Pinard inc.
 # François Pinard <pinard@iro.umontreal.ca>, 2002.
 
--- a/Pymacs/pymacs.py
+++ b/Pymacs/pymacs.py
@@ -1,3 +1,4 @@
 #!/usr/bin/env python
+# -*- coding: latin-1 -*-
 # Copyright © 2001, 2002, 2003 Progiciels Bourbeau-Pinard inc.
 # François Pinard <pinard@iro.umontreal.ca>, 2001.
